CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -591,8 +591,8 @@ CF_PRIVATE void _CF_dispatch_once(dispatch_once_t *, void (^)(void));
 #endif
 
 #if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_EMBEDDED_MINI
-CF_PRIVATE _Atomic(uint8_t) __CF120293;
-CF_PRIVATE _Atomic(uint8_t) __CF120290;
+CF_PRIVATE exxtern _Atomic(uint8_t) __CF120293;
```

Comment:
This cannot be right. Likewise, `CF_PRIVATE` must imply `extern`.

Reply:
Okay, so we should just change the definition of `CF_PRIVATE` then?